binary_converter.py

```diff
@@ -0,0 +1,75 @@
+import struct
+import io
+import numpy as np

```

Comment:
Just giving a note to myself that numpy is not part of standard python so people will need to install numpy if they don't already have it.  I'm not really against this since numpy is pretty essential but a few people had trouble installing numpy in the past.